donate/templates/payment/card.html

```diff
@@ -15,24 +15,25 @@
                 <div class="notification notification--neutral">
                     <div class="notification__container">
                         <div class="notification__message">
-                            {% trans "You are donating" %} {{ currency_info.symbol }} {{ form.amount.value|floatformat:"-2" }}
+                            {% trans "You are donating" %} {{ currency_info.symbol }} <span class="js-donation-value">{{ form.amount.value|floatformat:"-2" }}</span>
                         </div>
                         <div class="notification__action">
-                            <a href="#donate__amount-edit" data-amount-toggle>{% trans "Change amount" %}</a>
+                            <a href="#donate__amount-edit" class="donation-update__toggle" data-amount-toggle>{% trans "Change amount" %}</a>
                         </div>
                     </div>
                 </div>
 
                 <div class="donation-update" data-amount-actions>
-                    <div class="donation-update__container">
-                        <div class="form-item">
-                            <label for="donation-update">{{ currency_info.symbol }}</label>
-                            <input type="number" name="donation-update" value="{{ form.amount.value }}" step="1" min="1">
+                    <form class="donation-update__container">
+                        <div class="donation-update__form-item form-item">
+                            <label class="donation-update__label form-item__label" for="donation-update">{% trans "Donation Amount" %}</label>
+                            <div class="donation-update__currency-symbol">{{ currency_info.symbol }}</div>
+                            <input class="donation-update__input form-item__input" id="js-update-donation-value" type="number" name="donation-update" value="{{ form.amount.value }}" step="1" min="1">
```

Comment:
I think the `step` attribute will be problematic as it prevents setting decimal amounts, which they need the ability to do - e.g., $3.14 on Pi day. Maybe we just set `min` and round the value to 2 decimal places in JS?

Reply:
Seems like step isn't an issue, this is simple how many to go up when the number input's arrows are pressed. The user can still enter 3.14 if they want, I've added some js to ensure the number can only be up to 2 decimal places though.